cmd/managementd/main.go

```diff
@@ -65,6 +65,8 @@ func main() {
 	router.HandleFunc("/speaker", managementinterface.SpeakerTestHandler).Methods("GET")
 	router.HandleFunc("/speaker/status", managementinterface.SpeakerStatusHandler).Methods("GET")
 	router.HandleFunc("/disk-memory", managementinterface.DiskMemoryHandler).Methods("GET")
+	router.HandleFunc("/view-location", managementinterface.SetLocationHandler).Methods("GET", "POST") // Form to view and/or set location manually.
+	router.HandleFunc("/location", managementinterface.LocationHandler).Methods("POST")                // Set location via a POST request.
```

Comment:
Make the POST endpoint sit under `/api` (see just below this) and then make the user interface part be served from `/location`.